addons/web/__manifest__.py

```diff
@@ -191,6 +191,7 @@
             'web/static/src/webclient/icons.scss', # variables required in list_controller.scss
             'web/static/src/views/**/*',
             'web/static/src/webclient/**/*',
+            'web/static/src/scss/select2_overriden.scss',
```

Comment:
Yes, that's the bundle.... but I'm now thinking that adding the file here will rise a lot of red flags across the entire department ^^
Sorry for the headache , could you just add the code in webclient.scss ? In retrospective it's the best option